### ISSUE TITLE
Add CLI command to show a vault

### ISSUE THREAD

**Issue by adrianbenavides** [build-trust/ockam#3933]
Depends on: https://github.com/build-trust/ockam/issues/3932

Add a new command to show a vault given its name. It will show the vault name, its type (kms or ockam) and a list of the identities that it contains (this can be obtained by parsing the identities' vault_name field).

You can get a vault through the `CliState` from the `CommandGlobalOpts`: `opts.state.vaults.get(name)`. Same for identities.

---

_We love helping new contributors!_
_If you have questions or need help as you work on your first Ockam contribution, please leave a comment on [this discussion](https://github.com/ockam-network/ockam/discussions/1081)._
_If you're looking for other issues to contribute to, checkout [this discussion](https://github.com/build-trust/ockam/discussions/3206) and labels -  https://github.com/build-trust/ockam/labels/good%20first%20issue or https://github.com/build-trust/ockam/labels/help%20wanted_

**Comment by miles170** [comment]
I'm working on this and have a few questions.

1. how to create `identity` for vault?
2. what do you mean by "this can be obtained by parsing the identities' vault_name field"? I cannot find the `vault_name` field for identity.

**Comment by miles170** [comment]
> I'm working on this and have a few questions.
> 
>     1. how to create `identity` for vault?
> 
>     2. what do you mean by "this can be obtained by parsing the identities' vault_name field"? I cannot find the `vault_name` field for identity.

@adrianbenavides Would you mind assisting me? thanks!

**Comment by adrianbenavides** [comment]
Hey @miles170 ! Yeah, I have to rewrite the description of the issue. We discussed in [another issue](https://github.com/build-trust/ockam/pull/3930#issuecomment-1354843501) how we want to approach this. Here are the main details.

> how to create identity for vault?

You don't have to create any identity. The command will just retrieve the vault given its name and type. The `name` command should be optional, and if not passed, it will show the `default` vault.

> what do you mean by "this can be obtained by parsing the identities' vault_name field"? I cannot find the vault_name field for identity.

Forget about this for now, we won't show any information about the identity.
